src/components/grid-aware/ImageHeader/ImageHeader.module.css

```
@@ -90,6 +90,23 @@
   margin-top: 20px;
 }
 
+.descriptionWrapper :first-child {
+  margin: 0;
+}
+
+.descriptionWrapper :not(:first-child):not(li) {
+  margin-top: 15px;
+}
+
+.descriptionWrapper ul {
+  margin-left: 2em;
+}
+
+.descriptionWrapper ul li {
```

You may want to scope the selector so that we only select `li` elements that are the direct children of `ul` elements; otherwise, you may end up selecting things that are arbitrarily deep, such as in an `ol` that's under a `ul`.

```suggestion
.descriptionWrapper ul > li {
```